content/en/docs/tasks/administer-cluster/separate-mount-namespace.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Managing a Separate Mount Namespace for Kubernetes"
+linkTitle: "Separate Mount Namespace"
+weight: 100
+description: >-
+     Discussion and instructions on how to administer a Kubernetes system with all container-specific mountpoints in a separate mount namespace from the parent system
+---
+
+# Managing a Separate Mount Namespace for Kubernetes
+
+## What does this mean?
+
+Originally, Kubelet and the Container Runtime all ran in the same mount namespace as
+the host OS (systemd, login shells, system daemons).
+
+![Fig 1: Original mount propagation](images/original-k8s-mount-propagation.png)
+
+_Fig 1: Original mount propagation_
```

Comment:
nit: this would work well using a [figure](https://gohugo.io/content-management/shortcodes/#figure)

Reply:
I'll give that a shot - However, I didn't see my images in the local preview, just broken image links.  Do you know if I'm putting them in the wrong place?